framework/framework.go

```diff
@@ -97,13 +98,43 @@ func (f *Framework) ProcessCreate(obj interface{}, resources ...Resource) error
 	return nil
 }
 
+// ProcessCreateWithBackoff is the same as ProcessCreate but takes an additional
+// backoff factory returning new backoff implementations to retry the creation
+// on resource errors. Retries are implemented using a retry resource that wraps
+// each given resource. Errors are logged eventually in case an appropriate
+// logger is configured in the operator framework.
+func (f *Framework) ProcessCreateWithBackoff(obj interface{}, resources []Resource, backoffFactory func() backoff.BackOff) error {
+	var retryResources []Resource
+
+	for _, r := range resources {
+		resourceConfig := DefaultRetryResourceConfig()
+		resourceConfig.BackOff = backoffFactory()
+		resourceConfig.Logger = f.logger
+		resourceConfig.Resource = r
+
+		retryResource, err := NewRetryResource(resourceConfig)
```

Comment:
It would also be good to suggest a better solution instead of only pointing out what is "wrong".